spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/deployer/strategies/HandleHealthCheckStep.java

```diff
@@ -131,9 +133,11 @@ private void deleteExistingRelease(Release existingRelease, List<String> applica
 			this.deleteStep.delete(existingRelease, existingAppDeployerData, applicationNamesToUpgrade);
 		}
 		catch (DataAccessException e) {
+			logger.debug("Error1 deleteExistingRelease {}", e);
 			throw e;
 		}
 		catch (Exception e) {
+			logger.debug("Error2 deleteExistingRelease {}", e);
```

Comment:
Something better than `Error2` ... also same comment wrt to completely suppressing